packages/wonder-blocks-popover/src/components/popover.tsx

```diff
@@ -101,6 +102,20 @@ type Props = AriaProps &
          * Whether to show the popover tail or not. Defaults to true.
          */
         showTail: boolean;
+        /**
+         * Optional property to enable the portal functionality of popover.
+         * This is very handy in cases where the Popover can't be easily
+         * injected into the DOM structure and requires portaling to
+         * the trigger location.
+         *
+         * Set to "true" by default.
+         *
+         * CAUTION: Turning off portal could cause some clipping issues
+         * especially around legacy code with usage of z-indexing,
+         * Use caution when turning this functionality off and ensure
+         * your content does not get clipped or hidden.
```

Comment:
Thanks for adding the "Caution" section! 🙏. Could you please add this to `popover.argtypes.ts` as well?

It's unfortunate that we have to duplicate these comments, but we have to do it due to some issues with `react-docgen` not being able to extract the code comments when we use union types.

You can see how the `portal` docs don't show up here: https://5e1bf4b385e3fb0020b7073c-wphvlpzdmq.chromatic.com/?path=/docs/packages-popover-popover--docs

<img width="992" alt="Screenshot 2024-07-17 at 6 32 33 PM" src="https://github.com/user-attachments/assets/4e02b5f4-9d32-408e-ac49-91939de1298a">